pyslackers_website/core/static/js/member_map.js

```diff
@@ -0,0 +1,418 @@
+        'Europe/Tirane': ['41.33333', '19.83333'],
+        'Europe/Ulyanovsk': ['54.33333', '48.4'],
+        'Europe/Uzhgorod': ['48.61666', '22.3'],
+        'Europe/Vienna': ['48.21666', '16.33333'],
+        'Europe/Vilnius': ['54.68333', '25.31666'],
+        'Europe/Volgograd': ['48.73333', '44.41666'],
+        'Europe/Warsaw': ['52.25', '21'],
+        'Europe/Zaporozhye': ['47.83333', '35.16666'],
+        'Europe/Zurich': ['47.38333', '8.533333'],
+        'Indian/Chagos': ['-7.33333', '72.41666'],
+        'Indian/Christmas': ['-10.4166', '105.7166'],
+        'Indian/Cocos': ['-12.1666', '96.91666'],
+        'Indian/Kerguelen': ['-49.3527', '70.21750'],
+        'Indian/Mahe': ['-4.66666', '55.46666'],
+        'Indian/Maldives': ['4.166666', '73.5'],
+        'Indian/Mauritius': ['-20.1666', '57.5'],
+        'Indian/Reunion': ['-20.8666', '55.46666'],
+        'Pacific/Apia': ['-13.8333', '-171.733'],
+        'Pacific/Auckland': ['-36.8666', '174.7666'],
+        'Pacific/Bougainville': ['-6.21666', '155.5666'],
+        'Pacific/Chatham': ['-43.95', '-176.55'],
+        'Pacific/Chuuk': ['7.416666', '151.7833'],
+        'Pacific/Easter': ['-27.15', '-109.433'],
+        'Pacific/Efate': ['-17.6666', '168.4166'],
+        'Pacific/Enderbury': ['-3.13333', '-171.083'],
+        'Pacific/Fakaofo': ['-9.36666', '-171.233'],
+        'Pacific/Fiji': ['-18.1333', '178.4166'],
+        'Pacific/Funafuti': ['-8.51666', '179.2166'],
+        'Pacific/Galapagos': ['-0.9', '-89.6'],
+        'Pacific/Gambier': ['-23.1333', '-134.95'],
+        'Pacific/Guadalcanal': ['-9.53333', '160.2'],
+        'Pacific/Guam': ['13.46666', '144.75'],
+        'Pacific/Honolulu': ['21.30694', '-157.858'],
+        'Pacific/Kiritimati': ['1.866666', '-157.333'],
+        'Pacific/Kosrae': ['5.316666', '162.9833'],
+        'Pacific/Kwajalein': ['9.083333', '167.3333'],
+        'Pacific/Majuro': ['7.15', '171.2'],
+        'Pacific/Marquesas': ['-9', '-139.5'],
+        'Pacific/Midway': ['28.1999992', '-177.3499986'],
+        'Pacific/Nauru': ['-0.51666', '166.9166'],
+        'Pacific/Niue': ['-19.0166', '-169.916'],
+        'Pacific/Norfolk': ['-29.05', '167.9666'],
+        'Pacific/Noumea': ['-22.2666', '166.45'],
+        'Pacific/Pago_Pago': ['-14.2666', '-170.7'],
+        'Pacific/Palau': ['7.333333', '134.4833'],
+        'Pacific/Pitcairn': ['-25.0666', '-130.083'],
+        'Pacific/Pohnpei': ['6.966666', '158.2166'],
+        'Pacific/Port_Moresby': ['-9.5', '147.1666'],
+        'Pacific/Rarotonga': ['-21.2333', '-159.766'],
+        'Pacific/Tahiti': ['-17.5333', '-149.566'],
+        'Pacific/Tarawa': ['1.416666', '173'],
+        'Pacific/Tongatapu': ['-21.1666', '-175.166'],
+        'Pacific/Wake': ['19.28333', '166.6166'],
+        'Pacific/Wallis': ['-13.3', '-176.166']
+    }
+
+    var features = slack_member_tz_count.map(([timezone, memberCount]) => (
+        {
+            "geometry": {
```

Comment:
You don't have to fix this, but I would after merge if you don't: you're mixing quotes back and forth throughout this file - generally I like to see them remain consistent.

Reply:
Sorted and now enforceable through linting.